device/iot-device-samples/pnp-device-sample/Thermostat/src/main/java/samples/com/microsoft/azure/sdk/iot/Thermostat.java

```diff
@@ -38,6 +37,7 @@
         }
     }
 
+    // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json
```

Comment:
Looks like we are adding this file to our own repo as well. So why not point it to our version of it? This repo is not owned by us and it can change so it might be safer to point to our own. Or we don't have a copy on our end so that there is no multi master problem